server/main.py

```diff
@@ -224,9 +218,18 @@ async def index(request):
             f'gsutil cp {METADATA_PREFIX}.json {quote(output_dir)}/metadata.json'
         )
 
+        # if
```

Comment:
Instead of `chmod`ing, could we run `/bin/sh <script>` always?

Reply:
As discussed, `/bin/sh python-script.py` doesn't work. The file existence check is to make sure if we're running a script directly, we make it executable, otherwise if it's the executor (like `python` or `R`), it does nothing.